academy/4-my-own-chain/game-winner.md

```diff
@@ -12,18 +12,18 @@ tag: deep-dive
 Make sure you have all you need before proceeding:
 
 * You understand the concepts of [Protobuf](../2-main-concepts/protobuf.md).
-* Have Go installed.
-* The checkers blockchain codebase with the deadline field and its handling. You can get there by following the [previous steps](./game-deadline.md) or checking out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/game-deadline).
+* Go is installed.
+* You have the checkers blockchain codebase with the deadline field and its handling. If not, follow the [previous steps](./game-deadline.md) or check out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/game-deadline).
 
 </HighlightBox>
 
-To be able to terminate games you need to identify games that have already been terminated. A good field to add is one for the **winner**. It needs to contain:
+To be able to terminate games you need to identify games that have already been terminated. A good field to add is for the **winner**. It needs to contain:
 
-* The rightful winner of a game that reaches completion.
-* Or, the winner by forfeit, when a game is expired.
+* The winner of a game that reaches completion.
+* Or the winner _by forfeit_ when a game is expired.
 * Or a neutral value when the game is active.
 
-In this exercise, a draw is not handled and it would require yet another value to save in _winner_.
+In this exercise, a draw is not handled as it would require yet another value to save in _winner_.
```

Comment:
Actually, the draw is not handled because it is another complicated set of actions that have to be implemented. The fact that, after implementing a draw, there would be another value to put in _winner_ is purely accessory, this is not what is preventing us from implementing a draw.

```suggestion
In this exercise, a draw is not handled, and as side-note, it would require yet another value to save in _winner_.
```
@AndrewB9lab